nautobot/circuits/tests/test_filters.py

```diff
@@ -212,11 +212,11 @@ def test_provider_network(self):
         params = {"provider_network_id": [provider_network[0].pk, provider_network[1].pk]}
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
 
-    def test_type(self):
+    def test_circuit_type(self):
         circuit_type = CircuitType.objects.first()
-        params = {"type_id": [circuit_type.pk]}
+        params = {"circuit_type": [circuit_type.pk]}
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 3)
-        params = {"type": [circuit_type.slug]}
+        params = {"circuit_type": [circuit_type.slug]}
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 3)
```

Comment:
While you're touching this test case, would you mind changing it to use `assertQuerysetEqualAndNotEmpty`?